code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -49,6 +49,11 @@
 			icon = 'icons/mob/alien.dmi'
 			icon_state = "AlienMMI"
 			alien = 1
+		else if(istype(O,/obj/item/organ/internal/brain/drask))

```

Comment:
i don't expect you to change this



but istype lists for this sorta thing are aaaaaaaaaaaaaaaaaaaaaawful